website/blog/2024-10-23-NOVA/index.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Unlocking the Power of Agentic Workflows at Nexla with Autogen
+authors:
+  - Noel1997
+  - ameydesai
+tags: [data automation, agents, Autogen, Nexla]
+---
+
```

Comment:
Inserting an image here would increase the interest of readers. Optional.

Reply:
Added